modules/project-docs/pages/sdk-release-notes.adoc

```diff
@@ -29,11 +29,55 @@ Java 11 has various enhancements like lambda local variable type inference, prof
     <dependency>
         <groupId>com.couchbase.client</groupId>
         <artifactId>java-client</artifactId>
-        <version>3.1.0</version>
+        <version>3.1.1</version>
     </dependency>
 </dependencies>
 ----
 
+== Version 3.1.1 (12 January 2021)
+
+Version 3.1.1 is the second release of the 3.1 series, bringing stabilizations and enhancements over 3.1.0.
+
+http://packages.couchbase.com/clients/java/3.1.1/Couchbase-Java-Client-3.1.1.zip[Download] |
+https://docs.couchbase.com/sdk-api/couchbase-java-client-3.1.1/index.html[API Reference] |
+http://docs.couchbase.com/sdk-api/couchbase-core-io-2.1.1/[Core API Reference]
+
+The supported and tested dependencies for this release are:
+
+* com.couchbase.client:**java-client:3.1.1**
+* com.couchbase.client:**core-io:2.1.1**
+* io.projectreactor:**reactor-core:3.4.1**
+* org.reactivestreams:**reactive-streams:1.0.3**
+
+== API Affecting
```

Comment:
this has a different heading type than the others

Reply:
ooh good spot